### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Adamkaram**
adding links for available realses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,27 +5,22 @@
...
-    Windows
-    ·
-    Linux
-    ·
-    iOS
-    ·
-    watchOS
-    ·
-    Android
+   <a href="https://www.spacedrive.com/api/releases/desktop/stable/darwin/x86_64" >macOSIntel</a> ·
```
Comment (brxken128) [inline README.md:16]: MacOS has both ARMv8 (Aarch64, also known as Apple Silicon) and x86_64 builds, and the formatting is incorrect. You could use something such as this:

```markdown
<details>
    <summary>MacOS</summary>
    · <a href="https://www.spacedrive.com/api/releases/desktop/stable/darwin/aarch64">Apple Silicon</a>
    <br />
    · <a href="https://www.spacedrive.com/api/releases/desktop/stable/darwin/x86_64">Intel</a>
</details>
```

Or you could have two separate links, one for each, but using a different delimiter than `·`.

```markdown
MacOS - <a href="https://www.spacedrive.com/api/releases/desktop/stable/darwin/aarch64">Apple Silicon</a> | <a href="https://www.spacedrive.com/api/releases/desktop/stable/darwin/x86_64">Intel</a> ·
```

I think the second option would be preferable.

File: README.md [hunk 2]
```diff
@@ -5,27 +5,22 @@
...
+		   <a href="https://www.spacedrive.com/api/releases/desktop/stable/windows/x86_64" >Windows</a> ·
+   <a href="https://www.spacedrive.com/api/releases/desktop/stable/linux/x86_64" >Linux</a> ·
+     <a href="" >IOS</a> ·
+    <a href="" >watchOS</a> ·
+		 <a href="" >Android</a> ·
```
Comment (brxken128) [inline README.md:21]: These shouldn't be clickable as no releases are available.

File: README.md [hunk 3]
```diff
@@ -5,27 +5,22 @@
...
     <br />
-    <i>~ Links will be added once a release is available. ~</i>
+    <i>~ Links for (IOS · watchOS · Android ) will be added once a release is available. ~</i>
```
Comment (brxken128) [inline README.md:23]: `IOS` should be formatted as `iOS`, and `watchOS` as `WatchOS`.

File: README.md [hunk 4]
```diff
@@ -5,27 +5,22 @@
...
-    ·
-    iOS
-    ·
-    watchOS
-    ·
-    Android
+   <a href="https://www.spacedrive.com/api/releases/desktop/stable/darwin/x86_64" >macOSIntel</a> ·
+		   <a href="https://www.spacedrive.com/api/releases/desktop/stable/windows/x86_64" >Windows</a> ·
+   <a href="https://www.spacedrive.com/api/releases/desktop/stable/linux/x86_64" >Linux</a> ·
+     <a href="" >IOS</a> ·
```
Comment (mmattbtw) [inline README.md:19]:
```suggestion
     <a href="" >iOS</a> ·
```

correct capitalization :-)

File: README.md (hunk 1 above)
Comment (mmattbtw) [inline README.md:16]: imo do 
> macOS ([Intel]()/[Apple Silicon]()]

**Review by brxken128 — DISMISSED** [review]
The formatting is pretty inconsistent, in both tabs and OS names. I do agree that the download links should be added to the README now though.

File: README.md [hunk 5]
```diff
@@ -5,27 +5,22 @@
   <p align="center">
    <img width="150" height="150" src="packages/assets/images/AppLogo.png" alt="Logo">
   </p>
-  <h1 align="center"><b>Spacedrive</b></h1>
-  <p align="center">
+    <h1 align="center"><b>Spacedrive</b></h1>
+	 <p align="center">
```
Comment (mmattbtw) [inline README.md:9]: useless indentation change?

File: README.md (hunk 3 above)
Comment (mmattbtw) [inline README.md:23]:
```suggestion
    <i>~ Links for (iOS · watchOS · Android ) will be added once a release is available. ~</i>
```
correct capitalization :-)

**Review by mmattbtw — CHANGES_REQUESTED** [review]

**Comment by brxken128** [comment]
@mmattbtw I got you beat by just a second there LMAO

File: README.md (hunk 1 above)
Comment (Adamkaram) [reply README.md:16]: done

File: README.md (hunk 2 above)
Comment (Adamkaram) [reply README.md:21]: done

File: README.md (hunk 3 above)
Comment (Adamkaram) [reply README.md:23]: yes i didn't notice that sorry

**Review by Adamkaram — COMMENTED** [review]
everything is done now

**Review by Brendonovich — APPROVED** [review]

**Pull request merged**